lib/torrent.js

```diff
@@ -229,7 +230,7 @@ class Torrent extends EventEmitter {
       return this._destroy(new Error('Malformed torrent data: No info hash'))
     }
 
-    if (!this.path) this.path = path.join(TMP, this.infoHash)
+    if (!this.path) this.path = TMP
```

Comment:
I don't understand this change, why remove it, where is this being re added? in the store?

Reply:
the infoHash was only added to the path if the user didn't define his own path, and the infoHash was already passed in the store's name which I ended up using [here](https://github.com/feross/fs-chunk-store/pull/17/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R45)